main/src/ca/uwaterloo/flix/language/phase/unification/EffUnification3.scala

```diff
@@ -434,7 +434,7 @@ object EffUnification3 {
     implicit val renv: RigidityEnv = RigidityEnv.empty
     implicit val bimap: SortedBimap[Atom, Int] = mkBidirectionalVarMap(Atom.getAtoms(tpe))
 
-    val f0 = toSetFormula(tpe)(withSlack = false, scope, renv, bimap)
+    val f0 = toSetFormula(tpe)(withSlack = true, scope, renv, bimap)
```

Comment:
Found the first bug - slack variables were always deleted by simplify

Reply:
But thats OK? or?

Reply:
I guess I mean: NOT OK for type equivalence. OK if simplify is only used for printing.

What to do?

But I guess not documenting it is very bad.

Reply:
I don't think that it is okay, it's arbitrarily hiding information. Why would a simplifier hide anything from the type? `withSlack = true` just means "don't delete variables" which seems like the natural choice